From bf76a670ae82baab65465b9f38d9191873d72889 Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Thu, 22 Dec 2011 11:13:58 -0800
Subject: [PATCH 029/795] MR1: CLOUDERA-BUILD. Further tweaking Ivy resolver logic for MR1.
 (cherry picked from commit 4daeaa99b1c0733e9913740ea7017091ada9b074)

(cherry picked from commit 0ff31bb13ab1e49b5fa208b70f1398af3c6ee454)
---
 ivy/ivysettings.xml |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/ivy/ivysettings.xml b/ivy/ivysettings.xml
index de98674..90f88fe 100644
--- a/ivy/ivysettings.xml
+++ b/ivy/ivysettings.xml
@@ -111,7 +111,8 @@
     rather than look for them online.
 
     -->
-    <module organisation="org.apache.hadoop" name=".*" resolver="internal"/>
+    <!-- Commenting out to fix MR2 dependencies. -->
+    <!-- <module organisation="org.apache.hadoop" name=".*" resolver="internal"/> -->
     <!--until commons cli is external, we need to pull it in from the snapshot repository -if present -->
     <module organisation="org.apache.commons" name=".*" resolver="external-and-snapshots"/>
   </modules>
-- 
1.7.0.4

